_includes/codeTable.html

```diff
@@ -1,7 +1,7 @@
 <div class="row">
 
 <!-- Filter dropdowns -->
-<div class="col-md-3">
+<div class="col-md-2">
```

Comment:
I tried to decrease the div of the data table but it didn't work because I showed all the columns and they were too much.
Now some columns are hidden. I changed the div back to 3.